### DISCUSSION TITLE
Option to configure the lines between days in Event Mode

### DISCUSSION THREAD

**Discussion by herrzettpunkt** [totaldebug/atomic-calendar-revive#1366] [Feature Requests]
In the past, using the option `dayWrapperLineColor` in the Event Mode I could choose the background color to get rid of this line between the days but this option not available anymore. 

Is there a chance in the future that this option will come back or that you can deactivate this line using an option such as `showLineBetweenDays:` false ?

**Comment by marksie1988** [comment]
All color options were removed in v9 due to their being so many requests for different ones and it just adding far too many options to the card. 

Instead I now recommend using card_mod which I have explained how to use in this video https://www.youtube.com/live/-5MKd7LY-oc?si=R6zW6YtyWYdJTTOF

Almost all sections have classes associated with them, if it doesn't let me know and I can add one.

**Reply by herrzettpunkt** [reply]
Wow, the classes bring a whole new level of possibilities. Either I missed this in the documentation, or it might be worth mentioning as an additional way to customize your calendars. The YouTube video definitely opened my eyes - thank you very much for that!

**Reply by marksie1988** [reply]
Glad that's helped yea good idea I'll get the documentation updated on next release.
